substrate/zombienet/0001-basic-warp-sync/test-warp-sync.zndsl

```diff
@@ -13,18 +13,16 @@ charlie: reports peers count is at least 3 within 60 seconds
 dave: reports peers count is at least 3 within 60 seconds
 
 
-# db snapshot has 12133 blocks
-dave: reports block height is at least 1 within 60 seconds
-dave: reports block height is at least 12132 within 60 seconds
-dave: reports block height is at least 12133 within 60 seconds
+# db snapshot has {{DB_BLOCK_HEIGHT}} blocks
+alice: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
+bob: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
+charlie: reports block height is at least {{DB_BLOCK_HEIGHT}} within 60 seconds
```

Comment:
12333 check meant that alice,bob,charlie are building blocks. I'd prefer to keep this check. Please add also `greater then`?

Reply:
Sorry, you are right, in this test it is not needed. I see that some numbers could be messed, so we can fix numbers (or use `greater then`) in tests with validators. Thanks.